README.md

```diff
@@ -37,4 +37,44 @@ The configuration file follows a simple structure.
         ...
     ]
 }
-```
\ No newline at end of file
+```
+
+### Adding variable type
+
+You can specify the type of a variable by extending its name with the type in the query as such: ```variableName_variableType```. The underscore ```_``` here is crutial to make a clear distinction between name and type. 
+
+### Representation Mapper 
+
+If you want to add your own type representations you can do this by adding your representation to the [typeMapper.js](./src/typeMapper.js) file. 
+
+The mapper follows a simple structure:
+
+```js
+{
+    "typeName": mapperFunction,
+    ... 
+}
+```
+
+With ```typeName``` being the name of the variable as defined in the query and ```mapperFunction``` a function that takes the query result for the corresponding variable and returns either a string or a [React](https://react.dev/) component (see below).  
+
+Examples of how you can do this can already be found in that same file. 
+
+The web application uses [gridjs-react](https://gridjs.io/docs/integrations/react) internally. This allows us to add [html](https://nl.wikipedia.org/wiki/HyperText_Markup_Language) or [React](https://react.dev/) components as representations of the variable. To do this you need to import ```_``` function from [gridjs-react module](https://www.npmjs.com/package/gridjs-react) and call it with the component as its variable. An example for this is also already provided.
```

Comment:
- Newlines should be added after a sentence is finished.
- Add a link to the example.